lib/fs.js

```diff
@@ -470,7 +480,7 @@ function readFileAfterRead(err, bytesRead) {
 
 function readFileAfterClose(err) {
   var context = this.context;
-  var buffer = null;
+  var buffer;
```

Comment:
Without looking, my guess is that a default case/branch was missing at some point in the past, but that is no longer the case in this function.